### ISSUE TITLE
GPU-Driver Issue Deepin Linux Matebook e i3 Edtion 

### ISSUE THREAD

**Issue by florian-sabonchi** [linuxdeepin/developer-center#4211]
### SOFTWARE/OS VERSIONS | 软件/系统版本信息

```plain
Deepin 20.9
```

### PRE-REQUIREMENT | 预置条件

HUAWEI MateBook E, 12,6 Zoll 2-in-1 Laptop, Windows 11 S, 11. Generation Intel Core i3, 8GB RAM, 128GB SSD, 2,5K OLED Real Color FullView Display, Nebula Gray

### STEPS TO REPRODUCE | 复现步骤

1. Download Deepin Linux from Website iso
2. Install Deeplin Linux 
3. Deepin seems to work only in safe graphic mode "nomodeset" as Kernel Argument

**Comment by florian-sabonchi** [comment]
Seems like the issue is known and there alread an fix for that https://gitlab.freedesktop.org/drm/intel/-/issues/6476

**Comment by BLumia** [comment]
Does the issue still happen under deepin v23?

**Comment by florian-sabonchi** [comment]
> Does the issue still happen under deepin v23?

Yes, but I think this is an issue related to the kernel GPU driver

**Comment by florian-sabonchi** [comment]
> Does the issue still happen under deepin v23?

https://github.com/linuxdeepin/developer-center/issues/4211#issuecomment-1533940521 I found a workaround, you can find the fix here

**Comment by Zeno-sole** [comment]
This issue should be resolved in v23, but the new version of the GPU driver will have screen flickering issues in Matebook E 2022
